File: parallelogram.pas
var l, w, i ,tmp: integer;

begin
    write ( 'Input the height of parallelogram: ' );
    readln ( l );
    write ( 'Input the width of parallelogram: ' );
    readln ( w );
    
    for i := 1 to l do
    begin
        for tmp := l downto i do
            write ( ' ' );
        for tmp := 1 to w do
            write ( '@' );
        writeln;
    end;
end.

File: parallelogram.c
# include <stdio.h>

int main ( void )
{
    int l = 0, /* Line */
        w = 0, /* Width */
        i = 0,
        tmp = 0; /* Temp */
    
    printf ( "Input the height of parallelogram: " );
    scanf ( "%d", &l );
    printf ( "Input the width of parallelogram: " );
    scanf ( "%d", &w );
    
    for ( i = 1; i <= l; i++ )
    {
        for ( tmp = l; tmp >= i; tmp-- )
            printf ( " " );
        for ( tmp = 1; tmp <= w; tmp++ )
            printf ( "@" );
        printf ( "\n" );
    }
    return 0;
}
